bindings/generated/axelar-solana-its/src/program.ts

```diff
@@ -330,6 +330,11 @@ type AxelarSolanaIts = {
           isMut: false;
           isSigner: false;
         },
+        {
+          name: "gatewayRootPda";
+          isMut: false;
+          isSigner: false;
+        },
```

Comment:
Suprised about this change ! Do we maybe removed more than the account on previous pR :thinking: